### ISSUE TITLE
When I click the button "Hyper-parameter", jump to a blank page.

### ISSUE THREAD

**Issue by nnnnnzy** [microsoft/nni#4999]
**Describe the issue**:
When I click the button "Hyper-parameter", 

<img width="525" alt="1658173229464" src="https://user-images.githubusercontent.com/55190599/179603888-b444ca07-37b0-4170-b77b-ee6be37b9911.png">

jump to a blank page.

<img width="460" alt="1658173365696" src="https://user-images.githubusercontent.com/55190599/179604210-ef15e044-e0bd-408e-8b05-58b3e33dec65.png">

**Environment**:
- NNI version: 2.8
- Training service (local|remote|pai|aml|etc):remote
- Client OS:
- Server OS (for remote mode only):Ubuntu 18
- Python version:3.8.0
- PyTorch/TensorFlow version:Pytorch
- Is conda/virtualenv/venv used?:conda
- Is running in Docker?:no

**Configuration**:
 - Experiment config (remember to remove secrets!):

<img width="620" alt="1658173761648" src="https://user-images.githubusercontent.com/55190599/179605217-8542e192-9d76-4208-94d2-8691ce1312e8.png">

 - Search space:

<img width="247" alt="1658173785962" src="https://user-images.githubusercontent.com/55190599/179605251-d4f322a3-90e6-4b0a-8696-f0fba4629be1.png">

**Log message**:
 - nnimanager.log:

<img width="858" alt="1658173966870" src="https://user-images.githubusercontent.com/55190599/179605757-77b5cd72-6915-45f8-8c7a-768ee6bfc763.png">

 - dispatcher.log:
 
<img width="997" alt="1658173887208" src="https://user-images.githubusercontent.com/55190599/179605534-1a9c4d6d-73c3-4cc6-be9e-98104e31d258.png">

 - nnictl stdout and stderr:

stdout:
![Uploading 1658174219706.png…]()

stderr:
<img width="563" alt="1658174281182" src="https://user-images.githubusercontent.com/55190599/179606603-2e4ea836-633b-49bd-8ca6-3d4ea7804f33.png">

 

**How to reproduce it?**:

**Comment by Lijiaoa** [comment]
@nnnnnzy  Could you provide the trial data from this link? https://nni.readthedocs.io/en/stable/experiment/web_portal/web_portal.html#what-should-you-do-when-you-think-the-chart-is-strange-such-as-default-metric-hyper-parameter

**Comment by nnnnnzy** [comment]
Strangely enough, I found that changing to another port brought it back to normal. Thanks!

**Comment by scarlett2018** [comment]
Closing the issue as the problem has resolved.
